tests/testdata/expected/tofsee_440119c00bc748459e9c106e0847c2d83bd2aec3f5f57e1879e49baad7ec4f43.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
can you try to find a better config? There should be more complete ones somewhere. The same applies for other empty configs (except those that are empty intentionally)

Reply:
I need configs for tofsee, wshrat, zloader. The rest empty txt files are from igonored families so this is ok.

Reply:
Zloader and wshrat left...

Reply:
To be done in another PR.